leetcode/src/10.c

```diff
@@ -0,0 +1,25 @@
+bool isMatch(char* s, char* p);
```

Comment:
Could you please add small documentation about the problem?

Reply:
added documetation